packages/intent-filler/src/config/chains.toml

```diff
@@ -0,0 +1,44 @@
+[chains.bsc-chapel]
+type = "evm"
+chainId = 97
+stateMachineId = "EVM-97"
+wrappedNativeDecimals = 18
+rpcUrl = "BSC_CHAPEL"
+consensusStateId = "BSC0"
+
+[chains.bsc-chapel.assets]
+WETH = "0xb8c77482e45f1f44de1745f52c74426c631bdd52"
+DAI = "0x0000000000000000000000000000000000000000"
+
+[chains.bsc-chapel.addresses]
+UniswapV2Router = "0x0000000000000000000000000000000000000000"
+Handler = "0x4638945E120846366cB7Abc08DB9c0766E3a663F"
```

Comment:
Let's read the handler and uniswapv2 addresses directly from the ismp host contract `host_params` at runtime

Reply:
I have removed, fee token, handler, and univ2 addresses, and now using hostParams.